Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Lite_-_Incident_Handling.yml

```diff
@@ -16,6 +16,8 @@ description: |
 
   Verdict Handling:
   - Handles malicious incidents by initiating appropriate response actions, including blocking malicious indicators, isolating endpoints, and disabling user accounts.
+  
+  To utilize this playbook as the default for handling XDR incidents, the classifier should be empty, and the selected incident type should be `Cortex XDR - Lite`.
```

Comment:
what about the mapper here as well

Reply:
also add, In the integration configuration.....

Reply:
done